package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I'm probably being thicker than usual but why is there a package.json file in the root of the entire repository? Shouldn't we just need one in the Thali (and eventually Test) directories?